File: 0242-valid-anagram/0242-valid-anagram.c
bool isAnagram(char* s, char* t) {
    int n1 = strlen(s);
    int n2 = strlen(t);

    if (n1 != n2) {
        return false;
    }

    int freq[26] = {0}; 

    for (int i = 0; i < n1; i++) {
        freq[s[i] - 'a']++;
        freq[t[i] - 'a']--;
    }

    for (int i = 0; i < 26; i++) {
        if (freq[i] != 0) {
            return false;
        }
    }

    return true;
}

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
      char[] fc=s.toCharArray();
      char[] sc=t.toCharArray();
      Arrays.sort(fc);
      Arrays.sort(sc);
      String s1=new String(fc);
      String s2=new String(sc);
      if(s1.equals(s2)) 
        return true;
      else 
        return false;

    }
}